.github/workflows/ci_bindings_c.yml

```diff
@@ -47,17 +49,24 @@ jobs:
 
       - name: Setup Rust toolchain
         uses: ./.github/actions/setup
+
+
+      - name: Run sccache-cache
```

Comment:
Please remove this. OpenDAL intends not to introduce sccache for caching to ensure that opendal tests ghac correctly.